packages/modules/src/event/create/validateAndSanitizeCreateEvent.ts

```diff
@@ -0,0 +1,142 @@
+import moment from 'moment';
+
+import type { HandleCreateEventArgs } from './handleCreateEvent';
+
+async function validateAndSanitizeCreateEvent({ payload, context }: HandleCreateEventArgs) {
+  const { t } = context;
+  const { title, place, listAvailableAt, flyer, description, dateStart, dateEnd, classification, batches } = payload;
+
+  const data = {
+    title,
+    place,
+    listAvailableAt,
+    flyer,
+    description,
+    dateStart,
+    dateEnd,
+    classification,
+    batches,
+  };
+
+  if (!title) {
+    return {
+      ...data,
```

Comment:
why do you need to return data here if it's an error? maybe validate the input with yup?

Reply:
I just want to make sure I get the event data when calling the sanitizeAndValidatePayload function

I return same data always

Reply:
return data: null